WSDiscovery.py

```diff
@@ -1,6 +1,16 @@
 #!/usr/bin/env python
 
-import urllib.request, urllib.parse, urllib.error
+import sys
+
+if sys.version_info.major > 2:
+	import urllib.request, urllib.parse, urllib.error
+	urllib_parse_unquote = urllib.parse.unquote
```

Comment:
None of these imports are actually used except for unquote() from urllib.parse
Instead of making this alias, just use a from-style import and change the one usage in the code

    try:
        from urllib.parse import unquote
    except ImportError:
        from urllib2 import unquote